portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx

```diff
@@ -1099,11 +1121,19 @@ const ListObjects = ({
                   >
                     <UploadFolderIcon />
                   </BoxIconButton>
+                  <input
+                    type="file"
+                    multiple
+                    onChange={(e) => handleUploadButton(e)}
+                    id="file-input"
+                    style={{ display: "none" }}
+                    ref={folderUpload}
+                  />
                 </SecureComponent>
                 <input
                   type="file"
                   multiple
-                  onChange={(e) => uploadObject(e)}
+                  onChange={(e) => handleUploadButton(e)}
```

Comment:
```suggestion
                  onChange={handleUploadButton}
```
the arguments are same. so an extra function can be avoided